### ISSUE TITLE
Query parameters are not supported for repeated usage

### ISSUE THREAD

**Issue by a8t3r** [r2dbc/r2dbc-postgresql#122]
E.g. query:
```java
@Query("insert into users(id, name) values($1, $2) 
on conflict(id) do update set name = $2 returning *")
fun upsert(id: UUID, name: String): Mono<User>
```
contains parameter `$2` twice. Thus query will be failed on [Bindings.validate()](https://github.com/r2dbc/r2dbc-postgresql/blob/f8a358141089de9e47f4e73f672e2db7f6df1f3c/src/main/java/io/r2dbc/postgresql/client/Binding.java#L138) method because expected parameters are counted by occurrences of `\$[\d]+` regexp.

**Comment by Squiry** [comment]
I believe #111 fixes that.

**Comment by jurakontor** [comment]
Any idea when https://github.com/r2dbc/r2dbc-postgresql/pull/111/commits/c6926efba5cccd30282295152109185d8a20fae2 will be merged into codebase?

**Comment by LaoTsing** [comment]
please fix multiple usage of counters!

**Comment by mp911de** [comment]
Fixed via #111.
